### PR TITLE
website documentation

### PR DESCRIPTION
**Pull request by itkg-bfouche**

### CODE REVIEW

File: en/key_concepts.rst [hunk 1]
```diff
@@ -137,15 +137,18 @@ Open Orchestra's bundles :
...
-.. _Présentation des contenus:
-.. _Les paramètres d'un noeud:
-.. _configuration d'un template:
-.. _Présentation des types de contenus:
+.. _list of blocks:
+.. _node parameters:
+.. _configuring a zone:
+.. _configuring a template:
+.. _Presentation of contents:
+.. _Presentation of the Content Types:
```
Comment (juchi) [inline en/key_concepts.rst:157]: Some links can already be contributed

File: en/user_guide/websites_creation.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Website creation
+================
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+Name
+----
+
+The field name contributes the name of the website and he allows to generate the identifier of the website.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: the name field
it allows

File: en/user_guide/websites_creation.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+L'identifiant du site
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: translation

File: en/user_guide/websites_creation.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+---------------------
+
+The site id of field is generated when the name of the site is filled, but it remains alterable during the creation of the site.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: The "site id" field

File: en/user_guide/websites_creation.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+This attribute is unique, once the site is created it is not possible any more to modify it.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: not possible to modify it anymore

File: en/user_guide/websites_creation.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+Open Orchestra verifies during the form validation that the site id does not already exist.
+
+Aliases
+-------
+
+Open Orchestra allows to create  several alias for a site.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: aliases

File: en/user_guide/websites_creation.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+For example :
+
+* Scheme by default : http
+* Domain : demo.openorchestra.dev
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: i would rather put an url like www.example.com...

File: en/user_guide/websites_creation.rst [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+* Language : English
+* Language prefix : en
+* main aliases : on
+
+The website home page will be ``http://demo.openorchestra.dev/en``.
+
+Open Orchestra use only the main aliases for the sitemap generation.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: uses

File: en/user_guide/websites_creation.rst [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+Blocks available
+----------------
+
+This attribute list all the block available on Open Orchestra.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: lists
blocks
available for the nodes of this website ?

File: en/user_guide/websites_creation.rst [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+The selected blocks will be the ones visible during the node contribution.
+
+Theme by default
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: default theme ?

File: en/user_guide/websites_creation.rst (hunk 2 above)
Comment (alavieille) [inline en/user_guide/websites_creation.rst]: The field name is used for the website name and to generate the website identifier

File: en/user_guide/websites_creation.rst [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+----------------
+
+The attribute theme is the theme by default used for the website.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: default theme

File: en/user_guide/websites_creation.rst [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+It is possible to select an other theme during the node creation.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: another

File: en/user_guide/websites_creation.rst [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+sitemap generation
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: Capital S

File: en/user_guide/websites_creation.rst [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+------------------
+
+The two folling attribute are used for the sitemap generation :
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: attributes

File: en/user_guide/websites_creation.rst [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+~~~~~~~~~~
+
+This attribute defines whether or not the page should be referenced in search engines.
+
+Meta follow
+~~~~~~~~~~~
+
+This attribute indicates to search engines if they should follow the hypertext links on the page in order to index other documents.
+
+Informations contained in the robots.txt file
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: information

File: en/user_guide/websites_creation.rst [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+---------------------------------------------
+
+This attribute is what will contain the robots.txt of the website.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: is the data that the robots.txt file will contain

File: en/user_guide/websites_creation.rst [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+
+Example de contribution du robots.txt
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: translation

File: en/user_guide/websites_creation.rst (hunk 4 above)
Comment (alavieille) [inline en/user_guide/websites_creation.rst]: The site id can be generated from the site name or directly by the user at the creation.
In the remaining process, this field is not anymore editable

File: en/user_guide/websites_creation.rst [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    User-agent: *
+    Allow: /
+
+User-agent: * mean that one or several instruction (s) which follow applies for all the agents.
+Allow: mean that the search engine can browse all the directories and the pages of the site.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: Maybe explain separately what is Allow (list of paths that are crawlable), and "/" (all pages crawlable)

File: en/user_guide/websites_creation.rst [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+
+This attribute is what will contain the robots.txt of the website.
+
+Example de contribution du robots.txt
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    User-agent: *
+    Allow: /
+
+User-agent: * mean that one or several instruction (s) which follow applies for all the agents.
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: Maybe explain what is an agent (a crawler in this case)

File: en/user_guide/websites_creation.rst (hunk 9 above)
Comment (itkg-alavieille) [inline en/user_guide/websites_creation.rst]: available on Open Orchestra for the node contribution ?

File: en/user_guide/websites_creation.rst [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+
+The website home page will be ``http://demo.openorchestra.dev/en``.
+
+Open Orchestra use only the main aliases for the sitemap generation.
+
+Blocks available
+----------------
+
+This attribute list all the block available on Open Orchestra.
+The selected blocks will be the ones visible during the node contribution.
```
Comment (alavieille) [inline en/user_guide/websites_creation.rst]: will be visible during

File: en/user_guide/websites_creation.rst (hunk 17 above)
Comment (alavieille) [inline en/user_guide/websites_creation.rst]: translation

**Comment by itkg-bfouche** [comment]
@juchi @nicolasThal @itkg-nanne @itkg-ngilain

File: en/user_guide/websites_creation.rst [hunk 21]
```diff
@@ -0,0 +1,101 @@
...
+
+Open Orchestra uses only the main aliases for the sitemap generation.
+
+.. image:: ../../images/aliases.png
+
+Blocks available
+----------------
+
+This attribute lists all the blocks available for the nodes of this website.
+The selected blocks will be visible during the node contribution.
```
Comment (ngilain) [inline en/user_guide/websites_creation.rst]: Maybe something like "Only the selected blocks will be availables during the node contribution"

File: en/user_guide/websites_creation.rst [hunk 22]
```diff
@@ -0,0 +1,101 @@
...
+
+.. image:: ../../images/blocks_available.png
+
+Default Theme (required)
+------------------------
+
+The attribute theme is the default theme used for the website.
```
Comment (ngilain) [inline en/user_guide/websites_creation.rst]: used by the website ?

**Comment by itkg-bfouche** [comment]
@itkg-ngilain updated

**Comment by juchi** [comment]
:+1:

**Comment by itkg-bfouche** [comment]
@itkg-ngilain @itkg-nanne @nicolasThal

File: en/key_concepts.rst [hunk 23]
```diff
@@ -9,9 +9,10 @@ Nodes
 Nodes are mainly pages that the visitors can see on the website, they are containers for zones and blocks.
 There are two types of nodes in Open Orchestra:
 * Nodes that represent pages of the website
-* The transverse node (which is unique) containing the transverse blocks (blocks that are common to multiple pages of a website, for instance the menu block).
+* The transverse node (which is unique) containing the transverse blocks
```
Comment (nicolasThal) [inline en/key_concepts.rst]: The node is not unique, there is no version, but one per language

File: en/key_concepts.rst [hunk 24]
```diff
@@ -9,9 +9,10 @@ Nodes
...
+  (blocks that are common to multiple pages of a website, for instance the menu block).
```
Comment (nicolasThal) [inline en/key_concepts.rst]: blocks that can be used in other pages

File: en/user_guide/websites_creation.rst [hunk 25]
```diff
@@ -0,0 +1,101 @@
...
+This attribute lists all the blocks available for the nodes of this website.
+Only the selected blocks will be available during the node contribution.
+
+.. image:: ../../images/blocks_available.png
+
+Default Theme (required)
+------------------------
+
+The attribute theme is the default theme used by the website.
+It is possible to select another theme during the node creation.
```
Comment (nicolasThal) [inline en/user_guide/websites_creation.rst]: Each node can also have his own theme.

Comment (juchi) [inline en/user_guide/websites_creation.rst]: its own theme

**Comment by itkg-bfouche** [comment]
@juchi @nicolasThal

File: en/key_concepts.rst [hunk 26]
```diff
@@ -9,9 +9,10 @@ Nodes
 Nodes are mainly pages that the visitors can see on the website, they are containers for zones and blocks.
 There are two types of nodes in Open Orchestra:
 * Nodes that represent pages of the website
-* The transverse node (which is unique) containing the transverse blocks (blocks that are common to multiple pages of a website, for instance the menu block).
+* The transverse node (there is no version but one per language) containing the transverse blocks
```
Comment (juchi) [inline en/key_concepts.rst]: Sounds weird. I would remove the parenthesis here, and add a second sentence at the end
Something like "this node is not versionable, and there can only be once transverse node in each language of the site".
What do you think @nicolasThal @itkg-bfouche ?

Comment (nicolasThal) [inline en/key_concepts.rst]: I think that would be better

**Comment by itkg-bfouche** [comment]
@juchi @nicolasThal

**Comment by juchi** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1: after a little squash :)

**Pull request merged**